build/build_conda.sh

```
@@ -130,9 +122,9 @@ download_design_and_reference_files() {
     #     echo "Config directory not found, downloading config files"
     #     git clone --branch ${CONFIG_VERSION} ${CONFIG_GITHUB_REPO} hastings_config/
     # fi
-    if [ ! -d ${PIPELINE_NAME}_${TAG_OR_BRANCH} ]; then
+    if [ ! -d ${pipeline_dir} ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)